### ISSUE TITLE
Survey on the Quaternion Usage in Neural Network

### ISSUE THREAD

**Issue by hansen7** [RobinKa/tfga#16]
I think it is interested to think of the projects using TFGA for quaternion transformation with nn.

**Comment by RobinKa** [comment]
I agree, also here's the recent paper @hugohadfield talked about before https://arxiv.org/abs/2006.07360 which uses many different algebras that could probably be implemented easily using GA.

List of some relevant articles
- [AlgebraNets](https://arxiv.org/abs/2006.07360)
- [Quaternion Convolutional Neural Networks](https://arxiv.org/abs/1903.00658)
- [Quaternion Recurrent Neural Networks](https://arxiv.org/abs/1806.04418)
- [A theory of neural computation with Clifford algebras](https://www.semanticscholar.org/paper/A-theory-of-neural-computation-with-Clifford-Buchholz/0862074bcd040c7a6d967438539fa4feab29bd31)
- [A survey of quaternion neural networks](https://link.springer.com/article/10.1007/s10462-019-09752-1)

**Comment by hansen7** [comment]
add a few more on GNN,  3D Vision and Privacy:
- [NeurIPS2019] [Quaternion Knowledge Graph Embeddings](https://arxiv.org/pdf/1904.10281.pdf)
- [BMVC2018] [QuaterNet: A Quaternion-based Recurrent Model for Human Motion](https://arxiv.org/pdf/1805.06485.pdf)
- [ArXiv 2020] [Deep Quaternion Features for Privacy Protection](https://arxiv.org/pdf/2003.08365.pdf)
